CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # unreleased version
 
+# v4.1.0
+
+* Bump `slosilo` to v2.2 in order to be FIPS compliant
```

Comment:
Where is the `gemspec` change :)?

Reply:
It takes the latest slosilo therefore there is no explicit version

Reply:
If thats the case - don't see any reason for bumping version of `conjur-rack`

Reply:
Posum consumes conjur-rack with version